app/components/agenda/agenda-header.hbs

```diff
@@ -22,284 +22,201 @@
+                {{action "print"}}
+              >
+                {{t "print"}}
               </Menu.Item>
-              {{#if showPrintButton}}
-                <Menu.Item>
-                  <LinkTo
-                    @route="agenda.agendaitems"
-                    class="auk-button-link auk-button-link--block"
-                    data-test-agenda-header-navigate-to-agenda-view
-                  >
-                    <span class="auk-button__content">
-                      {{t "agenda-view"}}
-                    </span>
-                  </LinkTo>
-                </Menu.Item>
-                <Menu.Item>
-                  <a href=""
-                      data-test-agenda-header-print-agenda
-                      class="auk-button-link auk-button-link--block"
-                    {{action "print"}}
-                  >
-                    <span class="auk-button__content">
-                      {{t "print"}}
-                    </span>
-                  </a>
-                </Menu.Item>
-              {{else}}
-                <Menu.Item>
-                  <LinkTo
-                    data-test-agenda-header-navigate-to-printable-agenda
-                    class="auk-button-link auk-button-link--block"
-                    @route="agenda.print"
-                  >
-                    <span class="auk-button__content">
-                      {{t "printable-version"}}
-                    </span>
-                  </LinkTo>
-                </Menu.Item>
-              {{/if}}
-              {{#if currentSessionService.isEditor}}
+            {{else}}
+              <Menu.Item
+                @route="agenda.print"
+                data-test-agenda-header-navigate-to-printable-agenda
+              >
+                {{t "printable-version"}}
+              </Menu.Item>
+            {{/if}}
+            {{#if currentSessionService.isEditor}}
+              <Menu.Divider />
+              <Menu.Item
+                data-test-agenda-header-toggle-editing-session
+                {{action "toggleEditingSession"}}
+              >
+                {{t "edit-session"}}
+              </Menu.Item>
+              <Menu.Item
+                data-test-agenda-header-select-signature
+                {{action "selectSignature"}}
+              >
+                {{t "signature-edit"}}
+              </Menu.Item>
+            {{/if}}
+            <Menu.Item
+              data-test-agenda-header-download-documents
+              {{action "downloadAllDocuments"}}
+            >
+              {{t "download-documents"}}
+            </Menu.Item>
+            {{#if currentAgenda.isDesignAgenda}}
+              <Menu.Item
+                data-test-agenda-header-approve-all-agendaitems
+                {{action "showApproveAllAgendaitemsWarning"}}
+              >
+                {{t "approve-all-agendaitems"}}
+              </Menu.Item>
+            {{/if}}
+            {{#if currentSessionService.isEditor}}
+              {{#if
```

Comment:
Is deze conditional (om de divider correct te tonen, nagelang de items die al dan niet er onder komen te staan) nog ok om volledig inline te doen?